integration/helpers.go

```diff
@@ -1022,19 +1022,23 @@ type proxyServer struct {
 func (p *proxyServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// Validate http connect parameters.
 	if r.Method != http.MethodConnect {
-		trace.WriteError(w, trace.BadParameter("%v not supported", r.Method))
+		trace.WriteError(w, trace.Unwrap(trace.BadParameter("%v not supported", r.Method)))
```

Comment:
Not sure these changes are necessary - these are simply integration test helpers and it should not matter whether the strack traces are sent over the wire.